### PR TITLE
Covid resources

### PR DESCRIPTION
**Pull request by cmytelka**
Rough outline; I tried playing around with tailwind to get a different layout but I couldn't get it to work

### CODE REVIEW

File: src/components/nav/Nav.tsx [hunk 1]
```diff
@@ -83,6 +83,7 @@ const NavMenu = ({ open }: { open: boolean }) => {
         <NavItem href={"/"}>Conference Home</NavItem>
         <NavItem href={"/cfp"}>Call for Proposals</NavItem>
         <NavItem href={"https://ostem.org/"}>About oSTEM</NavItem>
+        <NavItem href={"/covid_resources"}>COVID-19 Resources</NavItem>
```
Comment (travigd) [inline src/components/nav/Nav.tsx:86]: Prefer dashes over underscores for URLS

File: src/pages/covid_resources.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import { Button } from "../components/button";
+
+![image](Images/rainbow.jpeg)
+
+# **COVID-19 Resources**
+
+### **Dear oSTEM Family,**
```
Comment (travigd) [inline src/pages/covid_resources.mdx:7]: Don't jump from h1 (`#`) to h3 (`###`)

File: src/pages/covid_resources.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+ 
+Academic and professional chapters are the foundation of oSTEM. Though community has often been defined by the value of in-person connections, we are committed to taking a stand against the public health risk caused by this pandemic.
+ 
+In support of your health and safety, until further notice, **we will cease all in-person oSTEM events and meetings and transition to a fully online community.**
+
+<Button onClick={"https://ostem.org/page/covid-19-response"}>
+  Read our Full Message
+</Button>
+
+### **oSTEM Squad Slack Workspace**
```
Comment (travigd) [inline src/pages/covid_resources.mdx:19]: We actually don't use squad slack, we use discord. We can maybe just change that later.

**Review by travigd — CHANGES_REQUESTED** [review]

File: src/pages/covid_resources.mdx (hunk 3 above)
Comment (cmytelka) [reply src/pages/covid_resources.mdx:19]: Will leave this section for now for future change to Discord info

**Comment by travigd** [comment]
It looks like this PR has combined the member and covid resources page at this point. For the sake of simplicity at this point, we can just use this PR for both and close #16.

**Pull request merged**